tests/src/it/scala/akka/kafka/TransactionsSourceSpec.scala

```diff
@@ -101,7 +119,6 @@ class TransactionsSourceSpec extends SpecBase
 
       val consumer = offsetValueSource(probeConsumerSettings(probeConsumerGroup), sinkTopic)
         .take(elements.toLong)
-        .idleTimeout(30.seconds)
```

Comment:
This is a magic number that can sometimes cause the test to abort too early before the transactional stream had a chance to complete.